code/datums/atom_huds/alternate_apperance.dm

```diff
@@ -239,6 +239,30 @@ var/global/list/active_alternate_appearances = list()
 		return TRUE
 	return FALSE
 
+// Fake-image can see only the specified faction
+/datum/atom_hud/alternate_appearance/basic/faction
+	var/datum/faction2check
+	add_ghost_version = TRUE
+
+/datum/atom_hud/alternate_appearance/basic/faction/New(key, image/I, faction)
+	..(key, I, FALSE)
+	if(SSticker)
+		faction2check = faction
+		var/datum/faction/F = find_faction_by_type(faction2check)
+		if(!F) return //Causes runtimes otherwise
```

Comment:
```suggestion
		if(!F)
			return //Causes runtimes otherwise
```

Reply:
Тут возможно подавление ошибки. Если F оказалось null-ом, то стоит разобраться, почему. Не вернул фракцию find_faction_by_type? Или конструктору передали не правильное имя faction? Или на момент вызова и создания худ-а для конкретного режима у нас почему-то не оказалось этого режима.

Reply:
Этот рантайм у меня вылетал когда я покупал фейк нюку без рольки синди, то-есть фракции ещё нет, а проверка есть. Отсюдого и проверка

Reply:
Тогда стоит дописать, что затычка именно для отсутствия гейммода, это "otherwise" очень расплывчато и кто-нибудь может потом запросто посчитать излишнем, проверить и убедиться что рантайма нет, и убрать проверку.

Еще не нажимай резолв комментариев сам, оставляй лучше эту кнопку на усмотрение ревьювера.